### PR TITLE
Optimized and cleaned up the snap

### PR DESCRIPTION
**Pull request by soumyaDghosh**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-summary: A small and simple sound and music player that is well integrated with GNOME.
-description: |
- Amberol aspires to be as small, unintrusive, and simple as possible. It does
- not manage your music collection; it does not let you manage playlists, smart
- or otherwise; it does not let you edit the metadata for your songs; it does
- not show you lyrics for your songs, or the Wikipedia page for your bands.
-
- Amberol plays music, and nothing else.
+adopt-info: amberol
+#summary: A small and simple sound and music player that is well integrated with GNOME.
```
Comment (alexmurray) [inline snapcraft.yaml]: Why have you commented out the summary and description?

File: snapcraft.yaml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+# or otherwise; it does not let you edit the metadata for your songs; it does
+# not show you lyrics for your songs, or the Wikipedia page for your bands.
+#
+# Amberol plays music, and nothing else.
 base: core22
 grade: stable
 confinement: strict
 license: GPL-3.0+
 compression: lzo
+icon: io.bassi.Amberol.svg
```
Comment (alexmurray) [inline snapcraft.yaml]: I would prefer not to ship a copy of the icon in the snap repo - can you explain why this is needed?

File: snapcraft.yaml [hunk 3]
```diff
@@ -47,27 +49,16 @@ parts:
     plugin: meson
     meson-parameters:
       - --prefix=/snap/amberol/current/usr
-    parse-info: [usr/share/metainfo/io.bassi.Amberol.xml]
     organize:
-      snap/amberol/current: .
+      snap/amberol/current/usr: usr
```
Comment (alexmurray) [inline snapcraft.yaml:44]: this change seems redundant - can you explain why it is needed?

File: snapcraft.yaml [hunk 4]
```diff
@@ -76,8 +67,8 @@ parts:
       - gstreamer1.0-plugins-good
       - gstreamer1.0-plugins-bad
       - gstreamer1.0-plugins-ugly
-      - libadwaita-1-0
-      - libdbus-1-3
+      #- libadwaita-1-0
```
Comment (alexmurray) [inline snapcraft.yaml]: instead of commenting out parts of the yaml these should just be removed if they are not needed anymore?

**Review by alexmurray — CHANGES_REQUESTED** [review]
Can you please add some more details on why you have made these changes?

File: snapcraft.yaml (hunk 1 above)
Comment (soumyaDghosh) [reply snapcraft.yaml]: Those are adopted from appdata, you can completely remove it

File: snapcraft.yaml (hunk 3 above)
Comment (soumyaDghosh) [reply snapcraft.yaml:44]: Yes. you can skip that too, That's not a big thing

File: snapcraft.yaml (hunk 2 above)
Comment (soumyaDghosh) [reply snapcraft.yaml]: It's easy to have a icon in the repo and done, it'll auto copy it in the snap store also and within the app, we'll not need to make the changes in the desktop file anymore

File: snapcraft.yaml (hunk 4 above)
Comment (soumyaDghosh) [reply snapcraft.yaml]: Okay, I am changing them

**Pull request closed without merging**